lib/proxy/class-iputils.php

```diff
@@ -0,0 +1,135 @@
+<?php
```

Comment:
Uups this will be harder to review. As part of the rules that needed some fixin the name of the file needed change. Not it looks like whole new blob. Probably it's better to change diff locally with git/your editor.

Reply:
Could split the changes up into one commit for the file name change (and related reference updates), and the other for the changes in the contents of this file. Then each commit can be viewed individually.

Reply:
Good idea! I thought it would make the git-hook we have there enforcing style not let me push, but it did!